src/grammar.rs

```diff
@@ -43,7 +43,7 @@ mod test {
   #[test]
   fn print_grammar_string() {
     let g :String = get_grammar_string();
-    info!("alles: {:?}", g);
+    println!("alles: {:?}", g);
```

Comment:
This seems very complicated, I leave it with `println!` for now, since `cargo` hides the output from successful tests.